src/test/EigenPod.t.sol

```diff
@@ -371,15 +362,6 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         require(address(pod).balance == 0, "Pod balance should be 0");
     }
 
-    function testAttemptedWithdrawalAfterVerifyingWithdrawalCredentials() public {
-        testDeployAndVerifyNewEigenPod();
-        IEigenPod pod = eigenPodManager.getPod(podOwner);
-        cheats.startPrank(podOwner);
-        cheats.expectRevert(bytes("EigenPod.hasNeverRestaked: restaking is enabled"));
-        IEigenPod(pod).withdrawBeforeRestaking();
-        cheats.stopPrank();
-    }
```

Comment:
perhaps the test here should be:

1. confirm 'hasRestaked = false'
2. prove withdrawal credentials
3. check that 'hasRestaked = false', as well as any other effects (like nonBeaconChainETH balance being sent and zeroed out)

Reply:
I dont think I follow - this is just trying to call funcstion that shouldn't be called once you've verified your pod WC

Reply:
mm OK. guess the suggested checks should go in `testDeployAndVerifyNewEigenPod` or similar then instead.